x-pack/filebeat/input/httpjson/chain.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// Example:
```

Comment:
Maybe would be good to have this examples in the docs if they are different from the current ones